security/acme-client/src/opnsense/mvc/app/models/OPNsense/AcmeClient/AcmeClient.xml

```diff
@@ -644,6 +645,20 @@
                 <dns_nsupdate_key type="TextField">
                     <Required>N</Required>
                 </dns_nsupdate_key>
+                <dns_opnsense_host type="TextField">
+                    <Required>N</Required>
+                    <default>localhost:443</default>
+                </dns_opnsense_host>
+                <dns_opnsense_key type="TextField">
+                    <Required>N</Required>
+                </dns_opnsense_key>
+                <dns_opnsense_token type="TextField">
+                    <Required>N</Required>
+                </dns_opnsense_token>
+                <dns_opnsense_insecure type="BooleanField">
+                    <Required>N</Required>
```

Comment:
should be mandatory - due to the default it should change to 0 in the migration

Reply:
Fixed the really mandatory fields